packages/vite-plugin-cloudflare/src/index.ts

```diff
@@ -198,15 +198,33 @@ export function cloudflare(pluginConfig: PluginConfig = {}): vite.Plugin {
 				}
 
 				config = workerConfig;
+
+				if (workerConfig.configPath) {
+					const dotDevDotVarsContent = getDotDevDotVarsContent(
+						workerConfig.configPath,
+						resolvedPluginConfig.cloudflareEnv
+					);
+					// if present we save a .dev.vars file onto the worker's build output
+					// directory so that it will be then detected by `vite preview`
```

Comment:
I find the following clearer

```suggestion
					// Save a .dev.vars file to the worker's build output directory 
					// when it exists so that it will be then detected by `vite preview`
```

So when I read those kind of comments, I only wonder and what if not???

Reply:
Oh well, looks like I misunderstood the comment... so maybe it wasn't clear :)